### PR TITLE
setup: added puppeteer for testing

### PR DESCRIPTION
**Pull request by ravikumar1002**
### What is the change?
 Did setup  of Puppeteer for testing

### Test report :
image

### CODE REVIEW

File: .github/workflows/percy.yml [hunk 1]
```diff
@@ -44,3 +44,4 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - run: npm install
       - run: npm run check
+      - run: npm run test
```
Comment (sahsisunny) [inline .github/workflows/percy.yml:47]:
```suggestion
```

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+coverage
```
Comment (sahsisunny) [inline .gitignore:2]:
```suggestion
coverage

```

File: .github/workflows/percy.yml (hunk 1 above)
Comment (ravikumar1002) [reply .github/workflows/percy.yml]: done

**Review by sahsisunny — APPROVED** [review]

**Review by vinit717 — APPROVED** [review]

File: __test__/index.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const timeout = 15000;
+
+describe('Dummy Test ', () => {
+  beforeAll(async () => {
+    await page.goto('https://www.realdevsquad.com/', {
```
Comment (rohan09-raj) [inline __test__/index.test.js:5]: Can we use this from constants ?

Comment (ravikumar1002) [reply __test__/index.test.js:5]: It's just a setup Demo,  we will remove it later. Is Need to replace it?

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**